src/test/java/org/dice_research/opal/launuts/polygons/parser/NutsParserMethodsTest.java

```diff
@@ -1,44 +1,64 @@
+		nutParser.geojsonReader = new FileReader("src/test/resources/nuts_synthetic_one/test-nuts-three.geojson");
+		JSONObject rootObject = (JSONObject) nutParser.jsonParser.parse(nutParser.geojsonReader);
+		JSONArray feature = (JSONArray) rootObject.get("features");
+		JSONObject firstObject = (JSONObject) feature.get(0);
+		JSONObject jsonGeometry = (JSONObject) firstObject.get("geometry");
+		JSONArray coordinates = (JSONArray) jsonGeometry.get("coordinates");
 
-//	@Test
-//	public void hasThisNutsLeastNumberOfCoordinatesIfTrueThenAdd() {
-//	}
+		// Read JSON array to compare with the result
+		nutParser.geojsonReader = new FileReader("src/test/resources/nuts_synthetic_one/test-nuts-four.geojson");
+		JSONObject rootObject1 = (JSONObject) nutParser.jsonParser.parse(nutParser.geojsonReader);
+		JSONArray feature1 = (JSONArray) rootObject1.get("features");
+		JSONObject firstObject1 = (JSONObject) feature1.get(0);
+		JSONObject jsonGeometry1 = (JSONObject) firstObject1.get("geometry");
+		JSONArray coordinatesExpected = (JSONArray) jsonGeometry1.get("coordinates");
+		JSONArray inner_rings = nutParser.getInnerRings(coordinates);
+		Assert.assertEquals("The result array returns inner-rings", coordinatesExpected, inner_rings);
+	}
 
 	@Test
-	public void getCoordinatesLatLongFormat() {
-		double[][] arrays = { { 8.81796, 50.13351 }, { 8.8237, 50.07466 }, { 8.71783, 50.06241 }, { 8.78065, 50.13673 },
-				{ 50.13351, 8.81796 } };
-		JSONObject coordinates = new JSONObject();
-		coordinates.put("coordinates", arrays);
-		JSONArray contentItems = new JSONArray();
-		contentItems.add(coordinates);
-		JSONArray lat_long = NutsParser.getCoordinatesLatLongFormat(contentItems);
-		System.out.println(lat_long);
+	public void getCoordinatesLatLongFormat() throws IOException, ParseException {
+		// Read JSON array to pass in the method
+		NutsParser nutsParser = new NutsParser();
+		nutsParser.geojsonReader = new FileReader("src/test/resources/nuts_synthetic_one/test-nuts.geojson");
+		JSONObject rootObject = (JSONObject) nutsParser.jsonParser.parse(nutsParser.geojsonReader);
+		JSONArray feature = (JSONArray) rootObject.get("features");
+		JSONObject firstObject = (JSONObject) feature.get(0);
+		JSONObject jsonGeometry = (JSONObject) firstObject.get("geometry");
+		JSONArray coordinates = (JSONArray) jsonGeometry.get("coordinates");
+
+		// Read JSON array to compare with the result
+		nutsParser.geojsonReader = new FileReader("src/test/resources/nuts_synthetic_one/test-nuts-two.geojson");
+		JSONObject rootObject1 = (JSONObject) nutsParser.jsonParser.parse(nutsParser.geojsonReader);
+		JSONArray feature1 = (JSONArray) rootObject1.get("features");
+		JSONObject firstObject1 = (JSONObject) feature1.get(0);
+		JSONObject jsonGeometry1 = (JSONObject) firstObject1.get("geometry");
+		JSONArray coordinatesExpected = (JSONArray) jsonGeometry1.get("coordinates");
+		JSONArray latLong = NutsParser.getCoordinatesLatLongFormat(coordinates);
```

Comment:
Move this line to upper block of code for better understanding:

JSONArray latLong = NutsParser.getCoordinatesLatLongFormat(coordinates);